dev-libs/log4cxx/log4cxx-0.10.0~git.recipe

```diff
@@ -6,12 +6,12 @@ notification levels."
 HOMEPAGE="https://logging.apache.org/log4cxx/latest_stable/index.html"
 COPYRIGHT="2004-2007 The Apache Software Foundation"
 LICENSE="Apache v2"
+srcGitRev="5f825186936a1876f92b88b371334ff26e997287"
 REVISION="1"
```

Comment:
Oops, you moved `REVISION` after `srcGitRev`...
```diff
+REVISION="1"
 srcGitRev="5f825186936a1876f92b88b371334ff26e997287"
-REVISION="1"
```
BTW, thanks for the switch to `srcGitRev` :)

Reply:
But you told me i have to follow this: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering

Reply:
Oops, I'm so sorry! That's my fault. I had not noticed that the wiki was wrong!!!

Reply:
I've just [fixed the wiki](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines/_compare/80e5f61%5E...80e5f61).

BTW, without your comment I would not have noticed that the wiki was wrong, so thanks for the heads-up :)